phone_number_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative "./phone_number"
+
+RSpec.describe PhoneNumber do
+  let(:phone_number) { PhoneNumber.new("5558675309") }
+
+  describe "#number" do
+    it "is a formatted number" do
+      expect(phone_number.number).to eq("(555) 867-5309")
+    end
+  end
+  
+  describe "#to_i" do
+    it "is the number as an int" do
+      new_number = PhoneNumber.new("555-867-5309")
+      expect(new_number.to_i).to eq(5558675309)
+    end
+  end
+
+  describe "#text" do
+    it "calls PhoneActions#text!" do
+      expect_any_instance_of(PhoneActions).to receive(:text!)
```

Comment:
It's good to make sure the method is getting called, but let's add some additional expectations to it. Specifically, we want to make sure it is texting the right number and sending the right message.

```suggestion
      expect_any_instance_of(PhoneActions).to receive(:text!).with("(555) 867-5309", "hello world!")
```

This would apply to the other specs that have expectations on PhoneActions.